doc/topics/releases/3006.rst

```diff
@@ -0,0 +1,123 @@
+|     OS   | Tiamat Path                                   | Non-Tiamat Path                         |
++==========+===============================================+=========================================+
+| RHEL     | https://repo.saltproject.io/salt/py3/redhat/  | https://repo.saltproject.io/py3/redhat/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Ubuntu   | https://repo.saltproject.io/salt/py3/ubuntu/  | https://repo.saltproject.io/py3/ubuntu/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Debian   | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Raspbian | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Fedora   | Hosted on Fedora Repos                        | Hosted on Fedora Repos                  |
++----------+-----------------------------------------------+-----------------------------------------+
+| MacOS    | https://repo.saltproject.io/salt/py3/osx/     | https://repo.saltproject.io/osx/        |
++----------+-----------------------------------------------+-----------------------------------------+
+| Windows  | https://repo.saltproject.io/salt/py3/windows/ |  https://repo.saltproject.io/windows/   |
++----------+-----------------------------------------------+-----------------------------------------+
+
+Note that the Tiamat paths above will not be available until the day of the Phosphorus release.
+
+
+My OS only provides Tiamat packages. What do I do?
+--------------------------------------------------
+If you are using a platform that no longer provides the non-Tiamat packages as shown
+in the previous tables, you will need to update your repo files to point to the new Tiamat packages.
+The new repo paths are provided in the previous tables, but the full instructions will be included
+at https://repo.saltproject.io.
+
+
+My OS uses an unsupported version of Python. What do I do?
+----------------------------------------------------------
+If your OS is using an unsupported version of Python, you will need to use the Tiamat packages.
+
+
+How do I migrate to the Tiamat packages?
+----------------------------------------
+The migration path from the non-Tiamat packages to the Tiamat packages will include:
+
+* Repo File: You need to update your repo file to point to the new repo paths for your platform. After the repo
+  file is updated, upgrade your Salt packages.
+* Pip packages: You need to ensure any 3rd party pip packages are installed in the correct Tiamat path.
+  This can be accomplished in two ways:
+  - `salt pip install <packag name>`
+  - Using the `pip.installed` Salt state.
```

Comment:
How will pip.installed be used to manage system pip packages?

Does this mean the default will now be installing pip packages inside salts own python? If so i can see this having a big effect on prebuilt formulas

Reply:
The `pip.installed` state will detect if its using tiamat packages or not. If its using tiamat packages it will install into the correct path for tiamat to see it, if not it will install is in the system path. 

I'm correct about that understanding right @s0undt3ch ^

Reply:
Thats my point. Thats effectively a "breaking" change from current. 
Any formulas / states trying to install pip packages at the system level will suddenly be installing inside tiamat's python

Reply:
If users are using the pip packages in order to use a Salt module, this would not be a breaking change, but I can see the scenario where someone intends on installing pip packages on the system level not intended to be used by Salt. 

@s0undt3ch can you comment here since you implemented the work here. What's the path for users to install at system level if they do not want to install inside the tiamat packages?

Reply:
In an internal discussion when we were implementing the work to move to tiamat packages, the consensus around what would be the most expecting behavior was, users tend to install python packages to support Salt modules, within the same state run, and that's the reasoning behind our decision.

In order to install python packages into the system python, and not into the tiamat bundled python, users will need to provide ``pip_bin`` or ``bin_env`` to the pip state module, for example:

```yaml
lib-foo:
  pip.installed:
    - pip_bin: /usr/bin/pip3

lib-bar:
  pip.installed:
    - bin_env: /usr/bin/python
```